client/config/webpack.common.js

```diff
@@ -57,12 +58,26 @@ module.exports = {
                         }
                     }
                 ]
+            }, {
+                test: /alloy-editor-no-react\.js$/,
+                use: {
+                    loader: 'imports-loader',
+                    options: {
+                        'React': 'react',
+                        'React.PropTypes': 'prop-types',
+                        'React.createClass': 'create-react-class'
```

Comment:
I assume a dependency needs this? Could you please document which one in a comment?

Reply:
`git blame client/config/webpack.common.js`:
```
696a4cb1 (Gertjan van Oosten 2018-04-03 16:31:32 +0200 61)             }, {
696a4cb1 (Gertjan van Oosten 2018-04-03 16:31:32 +0200 62)                 test: /alloy-editor-no-react\.js$/,
696a4cb1 (Gertjan van Oosten 2018-04-03 16:31:32 +0200 63)                 use: {
696a4cb1 (Gertjan van Oosten 2018-04-03 16:31:32 +0200 64)                     loader: 'imports-loader',
696a4cb1 (Gertjan van Oosten 2018-04-03 16:31:32 +0200 65)                     options: {
696a4cb1 (Gertjan van Oosten 2018-04-03 16:31:32 +0200 66)                         'React': 'react',
696a4cb1 (Gertjan van Oosten 2018-04-03 16:31:32 +0200 67)                         'React.PropTypes': 'prop-types',
696a4cb1 (Gertjan van Oosten 2018-04-03 16:31:32 +0200 68)                         'React.createClass': 'create-react-class'
696a4cb1 (Gertjan van Oosten 2018-04-03 16:31:32 +0200 69)                     }
696a4cb1 (Gertjan van Oosten 2018-04-03 16:31:32 +0200 70)                 }
```
`git log 696a4cb1`:
```
commit 696a4cb1fb0324982ed84b7f940b64ecb2ffe18b
Author: Gertjan van Oosten <gertjan@west.nl>
Date:   Tue Apr 3 16:31:32 2018 +0200

    NCI-Agency/anet#375: Re-add Alloy Editor
    
    And rewrite it such that the proper defines are there;
    rewrite is loosely based on https://github.com/liferay/alloy-editor/issues/770#issuecomment-355055340
```